hwp.py

```diff
@@ -46,6 +46,14 @@ def show_status(q_status: dict, t_status: dict) -> None:
 
 if __name__ == '__main__':
     args = sys.argv
+    if len(args) == 1:
+        print('''
+        setup                       -- setup login information
+        get <index>                 -- get question context
+        get all                     -- get all question list
+        submit <index> <filepath>   -- submit file
+        ''')
+        exit()
```

Comment:
if i remember correctly this would also print out whitspaces, i think this could be solved using `textwrap.dedent`